cms/djangoapps/contentstore/views/item.py

```diff
@@ -80,6 +81,14 @@
 ALWAYS = lambda x: True
 
 
+highlights_setting = WaffleSwitch(u'dynamic_pacing', u'studio_course_update')
+COURSE_UPDATE_WAFFLE_FLAG = CourseWaffleFlag(
+    waffle_namespace=WaffleFlagNamespace(name=u'schedules'),
+    flag_name=u'send_updates_for_course',
+    flag_undefined_default=False
+)
```

Comment:
Isn't this already defined here? https://github.com/edx/edx-platform/blob/43d6cc209b6e308c35dff2fc30ed6b7dda920944/openedx/core/djangoapps/schedules/config.py#L12